src/button/button-group.scss

```diff
@@ -61,11 +57,11 @@
           }
         }
       }
+    }
 
-      // Reset border radius only for non-borderless items
-      &:not(.iui-borderless) {
-        border-radius: 0;
-      }
+    // Reset border radius only for non-borderless items
+    #{$childSelector}:not(.iui-borderless) {
```

Comment:
This is interesting.  I see `$childSelector: 'input, button';` above.  Does that mean the SCSS is:
```scss
input, button:not(.iui-borderless) {
```

Or does it mean:
```scss
input:not(.iui-borderless),
button:not(.iui-borderless) {
```